src/relative-time.js

```diff
@@ -2,6 +2,8 @@
 
 import {strftime, makeFormatter, isDayFirst, isThisYear, isYearSeparator} from './utils'
 
+const rtf = new Intl.RelativeTimeFormat('en', {numeric: 'auto'})
+
 export default class RelativeTime {
```

Comment:
This `RelativeTime` class helper probably needs to say around to wrap are rules about which unit precision we prefer to display.